pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -2247,49 +2242,18 @@ public void removeNamespaceResourceGroup(@PathParam("tenant") String tenant,
     @ApiOperation(value = "Trigger the scan of offloaded Ledgers on the LedgerOffloader for the given namespace")
     @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission"),
             @ApiResponse(code = 404, message = "Namespace doesn't exist") })
-    public Response scanOffloadedLedgers(@PathParam("tenant") String tenant,
-            @PathParam("namespace") String namespace) {
+    public void scanOffloadedLedgers(@Suspended final AsyncResponse asyncResponse,
+                                     @PathParam("tenant") String tenant,
+                                     @PathParam("namespace") String namespace) {
         validateNamespaceName(tenant, namespace);
-        try {
-            StreamingOutput output = (outputStream) -> {
```

Comment:
why are you changing this ?
the result will be huge sometimes

Reply:
@hangc0276 requested this output to be served in a streaming way.
the result maybe really huge

Reply:
ok